cmd/check_container.go

```diff
@@ -152,111 +152,117 @@ var checkContainerCmd = &cobra.Command{
 			return err
 		}
 
-		// submitting results to pxysis if submit flag is set
+		// assemble artifacts and submit results to pyxis if user provided the submit flag.
 		if submit {
 			log.Info("preparing results that will be submitted to Red Hat")
-			log.Tracef("CertProject: %+v", certProject)
+			log.Tracef("CertProject: %+v", certProject) // TODO() This depends on Pyxis calls made outside of this block.
 
-			testResultsJsonFile, err := os.Open(path.Join(artifacts.Path(), certification.DefaultTestResultsFilename))
+			// The engine writes the certified image config to disk in a Pyxis-specific format.
+			// Include that data in our submission.
+			certImageFilepath := path.Join(artifacts.Path(), certification.DefaultCertImageFilename)
+			log.Tracef("Loading certImage data from path %s", certImageFilepath)
+			certImageBytes, err := os.ReadFile(certImageFilepath)
```

Comment:
I get that this reduces the complexity of the code and verbosity, and results in easier to read code, but don't we lose granularity of error handling in this case?

Reply:
Generally, I would agree, but we didn't handle `os.Open` and `io.Read` errors with any additional verbosity anyway. From my perspective, the operation we care about is "open and read" - if we can't do that, then `os.ReadFile`'s error should get report back to the user and should offer enough context.